src/pdn/src/grid_component.cpp

```diff
@@ -180,14 +180,14 @@ ShapePtr GridComponent::addShape(Shape* shape)
     const int y = std::min(static_cast<int>(die_area.yMin() + min_width),
                            final_shape_rect.yMax());
```

Comment:
Why do min with min_width?

Reply:
Just to give the shape a shape. And is consistent with the rest of the code